wgpu-hal/src/gles/egl.rs

```diff
@@ -50,16 +50,6 @@ type WlEglWindowResizeFun = unsafe extern "system" fn(
 
 type WlEglWindowDestroyFun = unsafe extern "system" fn(window: *const raw::c_void);
 
-#[cfg(target_os = "android")]
```

Comment:
An alternative would be to add `#[link(name = "android")]` here, but I think delegating this to `ndk-sys` is more robust.